internal/xds/translator/cluster.go

```diff
@@ -77,6 +78,30 @@ func buildXdsCluster(args *xdsClusterArgs) *clusterv3.Cluster {
 		cluster.TransportSocket = args.tSocket
 	}
 
+	for i, ds := range args.settings {
+		if ds.TLS != nil {
+			// for upstreamTLS , a fixed sni can be used. use auto_sni otherwise
```

Comment:
this comment is no longer valid, and can be removed

Reply:
done